newIDE/app/src/Profile/EditProfileDialog.js

```diff
@@ -160,6 +235,146 @@ const EditProfileDialog = ({
                 disabled={updateProfileInProgress}
                 floatingLabelFixed
               />
+              <LineStackLayout noMargin alignItems="center">
+                <Planet />
+                <TextField
+                  value={personalWebsiteLink}
+                  fullWidth
+                  translatableHintText={t`Personal website, itch.io page, etc.`}
+                  onChange={(e, value) => {
+                    setPersonalWebsiteLink(value);
+                  }}
+                  disabled={updateProfileInProgress}
+                  errorText={personalWebsiteLinkFormattingError}
+                  maxLength={150}
+                />
+              </LineStackLayout>
+              <LineStackLayout noMargin alignItems="center">
+                <Planet />
+                <TextField
+                  value={personalWebsite2Link}
+                  fullWidth
+                  translatableHintText={t`Personal website, itch.io page, etc.`}
```

Comment:
I feel like having exactly the same hint as for the field above gives a sense of a buggy interface. What do you think of `Another personal website, itch.io page, etc.`